spec/search_builders/hyrax/collection_member_search_builder_spec.rb

```diff
@@ -1,52 +1,89 @@
 RSpec.describe Hyrax::CollectionMemberSearchBuilder do
+  subject(:builder) { described_class.new(scope: context, collection: collection, search_includes_models: include_models) }
   let(:context) { double("context", blacklight_config: CatalogController.blacklight_config) }
   let(:solr_params) { { fq: [] } }
   let(:include_models) { :both }
   let(:collection) { build(:collection_lw, id: '12345') }
-  let(:builder) { described_class.new(scope: context, collection: collection, search_includes_models: include_models) }
 
   describe ".default_processor_chain" do
     subject { builder.default_processor_chain }
 
     it { is_expected.to include :member_of_collection }
+    it { is_expected.to include :filter_models }
   end
 
-  describe '#member_of_collection' do
-    let(:subject) { builder.member_of_collection(solr_params) }
+  context 'with a valkyrie collection' do
+    let(:collection) { build(:hyrax_collection, id: '12345') }
 
-    it 'updates solr_parameters[:fq]' do
-      subject
-      expect(solr_params[:fq]).to include("#{builder.collection_membership_field}:#{collection.id}")
+    describe '#member_of_collection' do
+      it 'updates solr_parameters[:fq]' do
+        expect { builder.member_of_collection(solr_params) }
+          .to change { solr_params[:fq] }
+          .to include("#{builder.collection_membership_field}:#{collection.id}")
+      end
     end
-  end
 
-  describe '#models' do
-    let(:work_classes) { [GenericWork] }
-    let(:collection_classes) { [Collection] }
-    let(:subject) { builder.models }
+    describe '#filter_models' do
```

Comment:
Somehow I feel like these names are backwards. I feel like #filter_models would filter out models and #include_models would include the models in the search. Im not sure if this was the way the interface was defined in hyrax originally, but something that stuck out here.